common.sh

```diff
@@ -469,7 +469,7 @@ if [[ ! -z ${AGENT_E2E_TEST_BOOT_MODE} ]]; then
 fi
 
 # For IPv6 (default case) mirror images are used since quay doesn't support IPv6
-if [[ -n "$MIRROR_IMAGES" || -z "${IP_STACK:-}" || "$IP_STACK" = "v6" || "$IP_STACK" = "v6v4" ]]; then
+if [[ -n "$MIRROR_IMAGES" || -z "${IP_STACK:-}" || "$IP_STACK" = "v6" ]]; then
```

Comment:
Reverting could broke https://github.com/openshift-metal3/dev-scripts/pull/1548, so it doesn't seem a good option. 
IMHO a better option would be to set `export MIRROR_IMAGES=true` when `IP_STACK` is set to `v6v4` in [network.sh](https://github.com/openshift-metal3/dev-scripts/blob/425b66a58cf266dafb7e3df8ce6aaf6b017d49d2/network.sh#L141) (and I guess jobs using it already explicitly set in the CI)

/cc @dtantsur @elfosardo

Reply:
Yep, export MIRROR_IMAGES=true will solve it but will also make deployment a disconnected one 

The deployment is working dine with MIRROR_IMAGES unset and v6 as primary in dualstack